.gitignore

```diff
@@ -2,4 +2,10 @@ node_modules
 bundle.js
 bundle.js.map
 bundle.js.LICENSE.txt
-.env
\ No newline at end of file
+.env
+resy-response.json
+resy-responses.json
+resy-responses_part1.json
+tomtom-response.json
+tomtom-responses.json
+tomtom-responses_part1.json
```

Comment:
These files should probably be in a folder. Then you could just gitignore the folder. But you'd need to update your seeds.rb so that it reads from the correct files still.